src/SecureHeaders.php

```diff
@@ -881,22 +875,114 @@ public function hpkproSubdomains($mode = true)
 
     public function done()
     {
-        $this->importHeaders();
-        $this->applyAutomaticHeaders();
+        // TODO: Move this elsewhere!
+        $directive = $this->canInjectStrictDynamic();
+        if ( ! is_int($directive)) {
```

Comment:
Should add `and is_string($directive)` to the initial `if` here (`canInjectStrictDynamic` will return the bool false if it the checks allowing it to inject fail, (integer return is an error), and string with the directive name if it can).

I think this was previously just `and $directive`, seeing now that probably wasn't clear enough ;p so `is_string` should do it?